### PR TITLE
ARC-1618 - Fix navigation from manage subscriptions page

### PR DESCRIPTION
**Pull request by krazziekay**
**What's in this PR?**
- Passed the `moduleKey` as a message from the child window to the parent window.
- Added an event listener to the parent, which checks the origin and the `moduleKey` and will navigate to that `moduleKey`.

**Why**
- Fixing routes.

**How has this been tested?**  
- Locally

### CODE REVIEW

File: static/js/github-configuration.js
Comment (gxueatlassian) [inline static/js/github-configuration.js:29]: /nit
maybe add a messageType along with the moduleKey, with 'goto_module_page'.
purely for code readability

**Review by gxueatlassian — APPROVED** [review]

File: static/js/github-configuration.js
Comment (krazziekay) [reply static/js/github-configuration.js:29]: I thought about adding it, but felt like its of not much value at this point. Will leave it for now...

**Comment by rachellerathbone** [comment]
Converted to draft because this needs to wait until Arc-1596 is merged.

**Pull request merged**
